algorithmic_efficiency/workloads/ogbg/ogbg_pytorch/models.py

```diff
@@ -0,0 +1,193 @@
+# Ported to PyTorch from
+# https://github.com/google/init2winit/blob/master/init2winit/model_lib/gnn.py.
+from typing import Callable, Optional, Tuple
+
+import jax.tree_util as tree
+from jraph import GraphsTuple
+import torch
+from torch import nn
+from torch_scatter import scatter
+
+
+def _make_mlp(in_dim, hidden_dims, dropout_rate):
+  """Creates a MLP with specified dimensions."""
+  layers = []
+  for dim in hidden_dims:
+    layers.extend([
+        nn.Linear(in_features=in_dim, out_features=dim),
+        nn.LayerNorm(dim),
+        nn.ReLU(),
+        nn.Dropout(dropout_rate)
+    ])
+  return nn.Sequential(*layers)
+
+
+class GNN(nn.Module):
+  """Defines a graph network.
+  The model assumes the input data is a jraph.GraphsTuple without global
+  variables. The final prediction will be encoded in the globals.
+  """
+  latent_dim: int = 256
+  hidden_dims: Tuple[int] = (256,)
+  dropout_rate: float = 0.1
+  num_message_passing_steps: int = 5
+
+  def __init__(self, num_outputs: int = 128) -> None:
```

Comment:
I changed it because the PyTorch GNN class needs a custom `__init__()` method, which leads to an error when trying to create a class instance by passing `num_outputs`, i.e. `model = GNN(num_outputs=128)`:
```
TypeError: __init__() got an unexpected keyword argument 'num_outputs'
```
Actually now that you are asking, I think generally it makes more sense to not use class attributes for all these arguments, since they are not necessarily meant to be the same for all `GNN` instances. Although for our benchmark they are, so I guess it's fine.